content-service/src/main/java/io/meeds/news/rest/NewsRest.java

```diff
@@ -821,4 +827,57 @@ public Response deleteArticleTranslation(@PathVariable("id")
       return Response.serverError().entity(e.getMessage()).build();
     }
   }
+
+  @GetMapping(path = "translation/{articleId}", produces = MediaType.APPLICATION_JSON_VALUE)
+  @Secured("users")
+  @Operation(summary = "Get article available translation languages", method = "GET", description = "Get article available translation languages")
+  @ApiResponses(value = { @ApiResponse(responseCode = "200", description = "article version deleted"),
+          @ApiResponse(responseCode = "400", description = "Invalid query input"),
+          @ApiResponse(responseCode = "401", description = "User not authorized to delete the article"),
+          @ApiResponse(responseCode = "500", description = "Internal server error") })
+  public ResponseEntity<List<String>> getAvailableTranslationLanguages(@PathVariable("articleId")
+                                                                       String articleId,
+                                                                       @RequestParam(name = "withDrafts")
+                                                                       boolean withDrafts) {
+    try {
+      if (StringUtils.isBlank(articleId)) {
+        return ResponseEntity.badRequest().build();
+      }
+      return ResponseEntity.ok(newsService.getArticleLanguages(articleId, withDrafts));
+    } catch (Exception e) {
+      LOG.error("Error when getting the article available translation languages with id " + articleId, e);
+      return ResponseEntity.internalServerError().build();
+    }
+  }
+
+  @GetMapping(value = "/languages", produces = "application/json")
```

Comment:
Could you add the documentation to this Rest function ?

Reply:
done!